packages/core/package.json

```diff
@@ -1,12 +1,35 @@
 {
   "name": "@friggframework/core",
   "prettier": "@friggframework/prettier-config",
-  "version": "0.2.30",
+  "version": "1.0.0",
+  "dependencies": {
+    "bcryptjs": "^2.4.3",
+    "mongoose": "6.11.6",
+    "jest-runner-groups": "^2.2.0",
+    "mongodb-memory-server": "^8.9.0",
```

Comment:
This and jest-runner-groups should go to devDependencies, yeah?

Reply:
yes